tests/common.rs

```diff
@@ -1,7 +1,8 @@
 extern crate laminar;
 
-use laminar::net::{constants, NetworkConfig, SocketAddr, UdpSocket};
+use laminar::net::{constants, NetworkConfig, UdpSocket};
 use laminar::packet::Packet;
+use std::net::SocketAddr;
```

Comment:
I don't like it because we don't provide the guarantees around it. To me it makes no sense to reexport it and just provides more confusion. Are we doing something to `SocketAddr` that I can't get by just using the one in `std`?

Reply:
No we don't do anything a with it. Most of the time if the users uses laminar::net it would also need std::net::SocketAddr. So I thought it would be nice to provide an re-export in our own net module. Must doesn't matter thayuch to me.